cypress/e2e/pages/testsForAllPages.js

```diff
@@ -28,6 +32,24 @@ export const testsThatAlwaysRunForAllPages = ({
       }
     });
 
+    it('all BBC links should contain a World Service', () => {
+      const allowedUrls = [
+        'programmes/p0703hz7', // present on https://www.bbc.com/persian/topics/cw9qgeqd1zqt & redirects to https://www.bbc.com/persian/podcasts/p0703hz7
+      ];
+
+      cy.get('main a[href^="https://www.bbc.com"]').each($tag => {
+        const href = $tag.attr('href');
+        expect(href).to.exist;
+        expect(href).to.not.be.empty;
+
+        const isAllowed = allowedUrls.some(url => href.includes(url));
+
+        if (!isAllowed) {
+          expect(href).to.match(`https://www.bbc.com/${allServices}`);
```

Comment:
`allServices` is an array of services, so we might need to create a regex in order to use the matcher
```suggestion
         expect(href).to.match(new RegExp(`https://www.bbc.com/${allServices.join('|')}*`));
```